mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
The z direction is fine, that's how I would've expected it to be defined. The final three numbers of the current MaxFilter point magnetometer definition (corresponding to the three normals) are 0, 0, 1.